### PR TITLE
Add new repo: ontove ontology

### PR DESCRIPTION
**Pull request by AlessandroBertozzi**

### CODE REVIEW

File: ontove/.htaccess [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Name of the project: Progetto Irnerio
+#
+# Description: The core part of this project consists of digitizing and classifying the vast collection 
+# of legal and theological-philosophical codices making up the rare-books collection of the Reale Collegio 
+# di Spagna in Bologna: with this digital catalogue, built under the guidance of Professors Domenico 
+# Maffei and Andrea Padovani, not only can the fragile codices be readily accessed and studied, but their 
+# preservation for future use is guaranteed, too. The project - named for the primus illuminator of 
+# legal science studies in Bologna, the Italian jurist Irnerius - have been possible by the Collegio di Spagna that
```
Comment (TallTed) [inline ontove/.htaccess:8]:
```suggestion
# legal science studies in Bologna, the Italian jurist Irnerius - have been made possible by the Collegio di Spagna that 
```

File: ontove/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The OntoVE project, Ontologies for the valorization of Germanic Cultural Heritage in Veneto within the perspective of the European Open Science Cloud (EOSC), has as its goal the development of  ontological modules for the description of Germanic Cultural Heritage (CH) items related to the Veneto region and provide different visualization outputs designed for both an academic and a non-academic audience.
```
Comment (TallTed) [inline ontove/README.md:1]:
```suggestion
The OntoVE project, Ontologies for the valorization of Germanic Cultural Heritage in Veneto within the perspective of the European Open Science Cloud (EOSC), has as its goal the development of ontological modules for the description of Germanic Cultural Heritage (CH) items related to the Veneto region and provision of different visualization outputs designed for both academic and a non-academic audiences.
```

File: ontove/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+The ultimate aim of the project is to connect heterogeneous data, such as archaeological findings or written documents, and provide uniform descriptions of them. The dataset is constituted by CH items of Germanic origin linked to Venice and its neighbouring territories. 
+In order to capture the defining features of these items, established ontologies are reused. The module devised for the description of CH items is called OntoVE_Archeo and reuses the top-level ontology CIDOC-CRM, v. 7.1.2 (Bekiari et al. 2022) and the finer-grained ontology ArCo-Architettura della Conoscenza, developed by Carriero et al. (2019) in collaboration with the ICCD (http://www.iccd.beniculturali.it/) and CNR (https://www.cnr.it/). 
+The OntoVE_WrittenRecords module has as its centre the semantic description of bibliographic items; it reuses LRMoo v 0.9.6 (Bekiari et al. 2023), Bibframe (https://www.loc.gov/bibframe/implementation/) and the Bibo (D’Arcus/Giasson 2016, https://www.dublincore.org/specifications/bibo/) ontology. The modules where developed with Protégé Desktop, version 5.5.0 (http://protege.stanford.edu, Musen 2015).
```
Comment (TallTed) [inline ontove/README.md:4]:
```suggestion
The OntoVE_WrittenRecords module has as its centre the semantic description of bibliographic items; it reuses LRMoo v 0.9.6 (Bekiari et al. 2023), Bibframe (https://www.loc.gov/bibframe/implementation/), and the Bibo (D’Arcus/Giasson 2016, https://www.dublincore.org/specifications/bibo/) ontology. The modules where developed with Protégé Desktop, version 5.5.0 (http://protege.stanford.edu, Musen 2015). 
```

File: ontove/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+The OntoVE_WrittenRecords module has as its centre the semantic description of bibliographic items; it reuses LRMoo v 0.9.6 (Bekiari et al. 2023), Bibframe (https://www.loc.gov/bibframe/implementation/) and the Bibo (D’Arcus/Giasson 2016, https://www.dublincore.org/specifications/bibo/) ontology. The modules where developed with Protégé Desktop, version 5.5.0 (http://protege.stanford.edu, Musen 2015). 
+Both modules align semantically equivalent classes and assign classes from finer-grained ontologies as subclasses of the more general and abstract classes provided by CIDOC-CRM and its interoperable bibliographic module LRMoo. By semantic alignment operations and the reuse of established ontologies the project adheres to the FAIR principles (Kush et al. 2020).
```
Comment (TallTed) [inline ontove/README.md:5]:
```suggestion
Both modules align semantically equivalent classes and assign classes from finer-grained ontologies as subclasses of the more general and abstract classes provided by CIDOC-CRM and its interoperable bibliographic module LRMoo. By semantic alignment operations and the reuse of established ontologies, the project adheres to the FAIR principles (Kush et al. 2020).
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small grammar and punctuation fixes

**Comment by dgarijo** [comment]
@AlessandroBertozzi please review suggestions and please clarify who the maintainer for this project will be. As stated in the htaccess it is:
```
# Contacts:
# - Monica Palmirani <monica.palmirani@unibo.it>
# - Silvio Peroni <silvio.peroni@unibo.it>
```
Is this correct? Please add yourself as a maintainer in the readme too

**Comment by AlessandroBertozzi** [comment]
I added the contacts and made the suggested changes to the texts. Sorry about the little mishap on the .htaccess. It was a text left over from an old project. I corrected and aligned it

**Pull request merged**